wallettool/src/main/java/org/bitcoinj/wallettool/WalletTool.java

```diff
@@ -83,7 +80,6 @@
 import java.time.LocalDate;
 import java.time.ZoneId;
 import java.time.format.DateTimeFormatter;
-import java.util.Collections;
 import java.util.List;
```

Comment:
The removal of unused imports is unrelated to these changes so should be in a separate commit. Good catch though.